src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working");

```

Comment:
The message in the `stopWork` method should also be more informative. Consider indicating that an `Excavator` has stopped working, to be consistent with the checklist requirements.